src/extended/tir_stream.c

```diff
@@ -536,6 +552,11 @@ static int gt_tir_searchforTIRs(GtTIRStream *tir_stream,
                                      seedptr->pos1 + seedptr->len);
         gt_seqabstract_reinit_encseq(sa_vseq, encseq, alilen,
                                      seedptr->pos2 + seedptr->len);
+        gt_log_log("reinit "GT_WU"-"GT_WU":"GT_WU"-"GT_WU,
+                seedptr->pos1 + seedptr->len,
+                seedptr->pos1 + seedptr->len + alilen,
+                seedptr->pos2 + seedptr->len,
+                seedptr->pos2 + seedptr->len + alilen);
```

Comment:
nitpicking to show that I actually look at the PRs: why didn't you indent to the ```(```?